pytest_selenium/pytest_selenium.py

```diff
@@ -194,10 +194,12 @@ def pytest_runtest_makereport(item, call):
                 _gather_html(item, report, driver, summary, extra)
             if 'logs' not in exclude:
                 _gather_logs(item, report, driver, summary, extra)
+            item.config.hook.pytest_selenium_runtest_makereport(
+                item=item, report=report, summary=summary, extra=extra)
```

Comment:
I believe this needs to be outside `if capture_debug` as it is a hook for other plugins to add details to the HTML report. They may want to do their own check for if debug information is desired, or enhance the report regardless of this condition. It's probably safe to do this outside the `driver is not None` check, as this can also be checked by anyone implementing the hook.